### ISSUE TITLE
Install fails on Ubuntu 20.04 release with gem dependency error

### ISSUE THREAD

**Issue by scdc-galvin** [jordansissel/fpm#1703]
FPM Install fails on a fresh Ubuntu 20.04 install.  When I run `sudo gem install fpm`, I get the following output:
```
ERROR:  Could not find a valid gem 'fpm' (>= 0), here is why:
          Unable to download data from https://rubygems.org/ - no such name (https://rubygems.org/specs.4.8.gz)
```

**Comment by scdc-galvin** [comment]
Re-ran the install after installing the latest round of updates for Ubuntu 20.04.  Fpm now installs, but gives the following message on use:
```
Doing `require 'backports'` is deprecated and will not load any backport in the next major release.
Require just the needed backports instead, or 'backports/latest'.
```

**Comment by AdolfVonKleist** [comment]
I can confirm I see the same problem on a fresh, straight-from-the-quickstart install on a fresh Ubuntu-18.04 install.  There the install goes fine initially, but the `backports` issue prevents actual usage just like @scdc-galvin mentions.

**Comment by BartKoppers** [comment]
Indeed, require just the needed backports instead. Using 'latest' however, is never recommended for production purposes. Should use semver specific packages/modules.
